src/util.h

```diff
@@ -118,17 +121,31 @@ namespace rhost {
             }
         };
 
+#ifdef _WIN32
         std::string Rchar_to_utf8(const char* buf, size_t len);
 
         inline std::string Rchar_to_utf8(const std::string& s) {
             return Rchar_to_utf8(s.data(), s.size());
         }
+        std::string from_utf8(const std::string& u8s);
+#else
+        inline std::string Rchar_to_utf8(const char* buf, size_t len) {
+            return std::string(buf, len);
```

Comment:
On any modern (like last several years) Linux distro it is. f it's not, we're in a world of hurt anyway because e.g. pathnames are not Unicode either.